src/sim-host/ui/sim-host.js

```diff
@@ -18,14 +20,17 @@ customElements.initialize(changePanelVisibilityCallback);
 
 window.addEventListener('DOMContentLoaded', function () {
     sizeContent();
-    Q.all([db.initialize(), initSocketPromise]).then(function () {
-        initializePlugins();
+    Q.all([db.initialize(), initSocketPromise]).then(function (result) {
```

Comment:
In fact  think I could simply replace `Q.all()` with `Q.spread()`. The reason I didn't is that `spread()` is Q specific, and not part of the Promise standard. While we currently use Q, I think it's preferable to work within the Promise standard if possible (which is why I also tend to avoid `Q.defer` these days).